assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -47,7 +48,10 @@ const Wrapper = styled.div`
 	${elementWithPosition}
 	${elementWithSize}
 	${elementWithRotation}
-	pointer-events: initial;
+  pointer-events: initial;
+  display: flex;
```

Comment:
This is to take advantage of flex's behavior and center the tooltip even though it's position absolute (instead of dealing with negative margins)

Reply:
I see. I like the general approach here, but there are some risks, since we don't know what else might end up inside the frame. One tricky case is the `text/frame.js` - it actually outputs the (opacity=0) text for some selection logic. Please check that doesn't get broken by flex here. I don't think it should, but it's slightly fragile. But quick clarification - why isn't this done in the `WithLink`?

Reply:
Ping, just in case we race-conditioned on Q&A.

Reply:
You're right, I moved it to `WithLink`